Gruntfile.js

```diff
@@ -84,6 +83,32 @@ module.exports = function(grunt) {
         command: './run_python_tests.sh'
       },
     },
+    
+    'grunt-chrome-build' : {
+      apprtc: {
+        options: {
+          buildDir: 'build/chrome-app',
+          zipFile: 'build/chrome-app/apprtc.zip',
+          chromeBinary: '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome',
+          keyFile: 'apprtc.pem'
```

Comment:
I think we'll store the .pem file internally. I'd leave comments here to say that the users of the demo app should fill in the chromeBinary and keyFile path.